.github/workflows/validation.yml

```diff
@@ -31,7 +31,8 @@ jobs:
       - run: |
           cd scripts
           cp ../target/release/{xline,benchmark,validation_lock_client} .
-          cp ../xline-test-utils/{private,public}.pem .
+          openssl genrsa -out ./private.pem
+          openssl rsa -in ./private.pem -pubout -outform PEM -out ./public.pem
```

Comment:
Maybe using a morden ECC algorithm like X25519 is better.

Reply:
The JWT token is transmitted in plaintext, and the key here is only used for signing to prevent the token from being tampered with.